src/lemlib/movements/turn.cpp

```diff
@@ -68,23 +68,22 @@ std::pair<int, int> Turn::update(Pose pose) {
     if (state == 1) return {128, 128};
 
     // reverse heading if doing movement in reverse
-    if (reversed) pose.theta = fmod(pose.theta - M_PI, 2 * M_PI);
+    if (reversed) pose.theta = units::mod(pose.theta - rot / 2, rot);
 
     // update completion vars
-    if (dist == 0) { // if dist is 0, this is the first time update() has been called
-        dist = 0.0001;
+    if (dist == 0_deg) { // if dist is 0, this is the first time update() has been called
+        dist = 0.0001_deg;
         startPose = pose;
     }
-    dist = fabs(radToDeg(angleError(pose.theta, startPose.theta)));
+    dist = angleError(pose.theta, startPose.theta);
 
     if (targetPose != std::nullopt) targetHeading = pose.angle(targetPose.value());
 
     // calculate error
-    float error = angleError(targetHeading, pose.theta);
+    Angle error = angleError(targetHeading, pose.theta);
 
     // calculate the speed
-    // converts error to degrees to make PID tuning easier
-    float output = angularPID.update(0, radToDeg(error));
+    float output = angularPID.update(0_deg, error); // todo: test
```

Comment:
```suggestion
    float output = angularPID.update(0_rad, error); // todo: test
```
Change to radians for consistency